README.md

```diff
@@ -40,10 +40,20 @@ Alternatively, dependencies can be installed into a [Python Virtual Environment]
     $ wget -nd http://www.tmbdev.net/en-default.pyrnn.gz
     $ mv en-default.pyrnn.gz models/
 
+An additional method using [Conda](http://conda.pydata.org/) is also possible:
+
+    $ conda create -n ocropus_env python=2.7
+    $ source activate ocropus_env
+    # PyPI and conda disagree on the name of the PyTables package:
```

Comment:
Good point about the rest of the README. It's a matter of taste I guess, not important.